static/app/components/events/contexts/index.tsx

```diff
@@ -92,11 +92,10 @@ export function EventContexts({event, group}: Props) {
   );
 
   useEffect(() => {
-    const transaction = Sentry.getActiveTransaction();
-    if (transaction && usingOtel()) {
-      transaction.tags.otel_event = true;
-      transaction.tags.otel_sdk = sdk?.name;
-      transaction.tags.otel_sdk_version = sdk?.version;
+    if (usingOtel()) {
+      Sentry.setTag('otel_event', true);
```

Comment:
This is probably not ideal because the tag will remain active for following navigation etc. as well 🤔 need to think about this. Does somebody know if this is still used, or if we could make this an attribute?

Reply:
changing this for attributes now, let's see if that's ok...